pkgs/data/themes/fluent-gtk-theme/default.nix

```
@@ -74,4 +74,4 @@ stdenvNoCC.mkDerivation rec {
     platforms = platforms.unix;
     maintainers = [ maintainers.romildo ];
```

Remove `with lib;` and use the components explicitly when needed.
For this particular line:
```suggestion
    maintainers = with lib.maintainers; [ romildo ];
```

The rest can be prepended with `lib.`

---

Done.